public/app/features/explore/Graph/ExploreGraph.tsx

```diff
@@ -69,7 +69,7 @@ export function ExploreGraph({
   onHiddenSeriesChanged,
   splitOpenFn,
   graphStyle,
-  tooltipDisplayMode = TooltipDisplayMode.Single,
```

Comment:
can we just remove this prop and cleanup if we are always going to use multi?

Reply:
Yeah, let's do that. New commit coming nooooow....